src/main/java/org/ndgf/endit/FlushTask.java

```diff
@@ -64,6 +81,18 @@ public FlushTask(FlushRequest request, Path outDir, String type, String name)
     @Override
     public Set<URI> start() throws IOException
     {
+        String checksumType="";
+        String checksumValue="";
+
+        for (Checksum checksum: checksums) {
+            checksumType = checksum.getType().getName().toLowerCase();
+            checksumValue = checksum.getValue();           
+        }
+
+        String s = String.format("{ \"file_size\": %d, \"time\": %d, \"storage_class\": %s, \"action\": %s, \"path\": %s, \"checksum_type\":%s, \"checksum_value\":%s }",
```

Comment:
you should use a standard tools/libraries, like gson to generate the json objects.

Reply:
I was just keeping track of the consistency of the source code. I mean a similar piece is also written in StageTask.java.
If it needs to be changed, then it needs to be changed everywhere (FlushTask.java and StageTask, java). 
If there is no objection from others (ndgf people), then I can make this change everywhere.

Reply:
Sure, but as we add more functionality, making it cleaner makes more sense.

Reply:
yes, right.
I am asking permission from ndgf as I am not the author of this plugin :).